cedar-drt/src/lean_impl.rs

```diff
@@ -180,7 +179,7 @@ impl LeanDefinitionalEngine {
                         // coerce to PolicyId just to ensure it's valid
```

Comment:
Oops, too quick with my merge - will try to remember to update next time I touch this code (this will likely be soon as I continue to muck with the FFI)